common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -120,4 +120,13 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
+  Switch(
+    ABTests,
+    "ab-tailor-survey",
+    "Integrate Tailor with ab tests",
+    owners = Seq(Owner.withGithub("oilnam")),
+    safeState = Off,
+    sellByDate = new LocalDate(2017, 3, 31),
```

Comment:
Do you mind setting this to a few days after the test expiry?  We're going to start trying to encourage this as best practice going forwards.